src/glaMemSup.jl

```diff
@@ -226,12 +232,14 @@ function GlaOprPrp(egoFur::AbstractArray{<:AbstractArray{T}}, trgVol::GlaVol,
 			# create Fourier transform plans
 			fftPlnFwd[dir] = plan_fft!(fftWrkFwd, [dir]; flags = FFTW.MEASURE)
 			fftPlnRev[dir] = plan_ifft!(fftWrkRev, [dir]; flags = FFTW.MEASURE)
+			adjFftPlnFwd[dir] = plan_fft!(fftWrkRev, [dir]; flags = FFTW.MEASURE)
+			adjFftPlnRev[dir] = plan_ifft!(fftWrkFwd, [dir]; flags = FFTW.MEASURE)
 		end
 	end
 	# computation of phase transformation
 	for itr ∈ eachindex(1:lvls)
 		# allows calculation odd coefficient numbers
-		phzInf[itr] = setTyp.([exp(-im * pi * k / brnSze[itr]) for 
+		phzInf[itr] = setTyp.([cis(-pi * k / brnSze[itr]) for 
```

Comment:
I forgot I added this, but the Julia docs claim [this method](https://docs.julialang.org/en/v1/base/math/#Base.cis) is more efficient than `exp(im*x)`